ONSdigital/go-ns
# Create a zebedee client for communication with zebedee server

### What
- Add a zebedee client to go-ns
- Unit tests for zebedee client
- Add zebedee page models

### How to review
- Verify the page models are correct
- Create a basic service which uses the client to test zebedee responses

### Who can review

Anyone
